### ISSUE TITLE
Clarity on documenting linguistic metadata would be helpful.

### ISSUE THREAD

**Issue by sofianef** [DOI-DO/dcat-us#179]
**Name:** Carrie Comstock

**Affiliation:** USEPA Office of Research and Development (ORD), Cummings.Carrie@EPA.gov

**Type of issue:** Schema, Editorial

Section # / Table #: [5.18.3](https://doi-do.github.io/dcat-us/#distribution-language) Distribution's _**language**_ property

**Issue:**

- **Substantive Comment**: There's a confusing usage note: "The property MUST be set if the distribution is language-dependent, or if it is given in some of the languages German, French, Italian and English but not in all four languages." It seems highly likely that some components of scientific data would be language-dependent and/or would be in English only, so this field would essentially be mandatory. It's also unclear why German, French, and Italian were the specific additional languages mentioned, given the U.S. context.

- **Rationale**: Clarity on documenting linguistic metadata would be helpful.

- **Proposed Disposition:** Removal or clarification of the sentence in question.

Original email submission: 
[USEPA-ScienceHub-FAIRness.Project.-.DCAT-US-3-AP.-.Comment.Review.Matrix.xlsx](https://github.com/DOI-DO/dcat-us/files/13430798/USEPA-ScienceHub-FAIRness.Project.-.DCAT-US-3-AP.-.Comment.Review.Matrix.xlsx)

**Comment by fellahst** [comment]
The following updates been done:

- Updated dcterms:language description for dcat:Catalog
- Updated dcterms:language description for dcat:CatalogRecord
- Updated dcterms:language description for dcat:DataService
- Updated dcterms:language description for dcat:Dataset
- Updated dcterms:language description for dcat:Distribution
- Added examples of dcterms:language in dcat:Dataset
- Update examples of dcterms:language in dcat:Distribution

**Comment by ShaferAC** [comment]
+1; 
Comment: There appears to be a ">" at the start of the Usage note about "Resources defined by the Library of Congress..." - should the arrow be removed?

In the dcterms:language description for dcat:Distribution - the 2nd and 3rd Usage notes seem a little redundant. I think the 3rd one is more concise and easier to understand. The dcterms:language for dcat:Dataset seems to use the same language as the 2nd Usage note in dcterms:language for dcat:Distribution. If it is decided to keep only the 3rd usage note, then we should probably update dcat:Dataset's dcterms:language usage note for consistency.

**Comment by mrratcliffe** [comment]
+1

**Comment by fellahst** [comment]
@ShaferAC,
 
Thank you for pointing out the typographical error and providing feedback on the usage notes in the DCAT-US profile. Your keen observation and suggestions are essential for maintaining the clarity and consistency of the documentation.

1. **Typographical Error**: We have addressed the issue with the stray ">" character at the start of the usage note about "Resources defined by the Library of Congress...".  We found 5 similar typos. This character was indeed not intended to be part of the text and has been removed for clearer readability.

2. **Usage Notes on `dcterms:language` for `dcat:Distribution` and `dcat:Dataset`**: After reviewing your comments regarding the redundancy in the usage notes, we agree that the third usage note is more concise and effectively communicates the necessary information. Therefore, we have removed the second usage note for `dcterms:language` under `dcat:Distribution` to eliminate redundancy.  This change will help maintain uniformity and clarity in the documentation.
